Apache-Beam-Jobs.md

```diff
@@ -670,6 +671,28 @@ The code above throws this error `'_UnwindowedValues' object is not subscriptabl
     ))
 ```
 
+### `_namedptransform is not iterable` error
+
+This error sometimes happens when you forget to add a label for some operation
+(the strings of code before `>>`).  The solution is to add a label for all operations. 
+
+#### Example
+
+```python
+some_values = (
+    some_models
+    | beam.Values()
+)
+```
+The code above might return `'_namedptransform is not iterable` in the job output.
+We can fix this by adding an appropriate label.
```

Comment:
ditto. This will be made into a single paragraph

Reply:
done